Formula/just.rb

```diff
@@ -16,6 +17,11 @@ class Just < Formula
 
   def install
     system "cargo", "install", *std_cargo_args
+
+    man1.install "#{buildpath}/man/just.1"
+    bash_completion.install "#{buildpath}/completions/just.bash" => "just"
+    fish_completion.install "#{buildpath}/completions/just.fish"
+    zsh_completion.install "#{buildpath}/completions/just.zsh" => "_just"
```

Comment:
```suggestion
    man1.install "man/just.1"
    bash_completion.install "completions/just.bash" => "just"
    fish_completion.install "completions/just.fish"
    zsh_completion.install "completions/just.zsh" => "_just"
```
Doesn't this work?

Reply:
Ah thanks, I must have goofed when making this.